dhis-2/dhis-e2e-test/src/test/java/org/hisp/dhis/tracker/deduplication/PotentialDuplicatesTests.java

```diff
@@ -93,35 +92,37 @@ public void shouldRequireBothTeis()
             .statusCode( equalTo( 400 ) )
             .body( "status", equalTo( "ERROR" ) )
             .body( "message", containsStringIgnoringCase( "missing required input property" ) );
+
+        potentialDuplicatesActions.createPotentialDuplicate( createTei(), null, "OPEN" )
```

Comment:
I'm wondering if this duplicate check is necessary or is it better to cover it in unit tests. e2e tests are generally time-consuming and checking both inputs instead of one might not add any meaningful coverage.